### PR TITLE
Fix for https://github.com/jfrog/charts/issues/12

### PR DESCRIPTION
**Pull request by jainishshah17**
Not bumping version as its going to be merged in the same PR https://github.com/helm/charts/pull/7320

### CODE REVIEW

File: stable/artifactory-ha/templates/_helpers.tpl [hunk 1]
```diff
@@ -10,14 +10,16 @@ Expand the name of the chart.
 The primary node name
 */}}
 {{- define "artifactory-ha.primary.name" -}}
-{{- printf "%s-%s-primary" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- $name := .Release.Name | trunc 15 -}}
```
Comment (rimusz) [inline stable/artifactory-ha/templates/_helpers.tpl]: to be sure we do have not too short release name, change to 29, tested it myself

File: stable/artifactory-ha/templates/_helpers.tpl [hunk 2]
```diff
@@ -10,14 +10,16 @@ Expand the name of the chart.
...
+{{- printf "%s-%s-primary" $name .Chart.Name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{/*
 The member node name
 */}}
 {{- define "artifactory-ha.node.name" -}}
-{{- printf "%s-%s-member" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- $name := .Release.Name | trunc 15 -}}
```
Comment (rimusz) [inline stable/artifactory-ha/templates/_helpers.tpl]: same here, change to 29

**Comment by jainishshah17** [comment]
@rimusz As you fixed this in chart-testing. Its already merged in the upstream chart. Do you still want me to change this?

**Comment by rimusz** [comment]
I'm not sure chart-testing fix will be accepted. yes fix that. with the next release to upstream it will be pushed there

**Comment by jainishshah17** [comment]
@rimusz done

**Pull request merged**